client/webserver/site/src/js/doc.ts

```diff
@@ -448,6 +449,15 @@ export default class Doc {
     el.textContent = msg
     Doc.show(el)
   }
+
+  // showFiatValue displays the fiat equivalent for the provided amount.
+  static showFiatValue (assetID: number, amount: number, display: PageElement): void {
+    const rate = app().fiatRatesMap[assetID]
+    if (rate) {
+      display.textContent = Doc.formatFiatConversion(amount, rate, app().unitInfo(assetID))
+      Doc.show(display.parentElement as Element)
+    } else Doc.hide(display.parentElement as Element)
+  }
```

Comment:
**doc.ts** shouldn't import `app`. This is off-pattern for these types of methods anyway. Try https://github.com/buck54321/dcrdex/compare/90ab9f5...buck54321:dcrdex:showfiatvalue

Reply:
Thanks. I cherry-picked your commit.